wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -316,6 +321,19 @@ deriveJSON Serokell.defaultOptions ''Address
 instance Arbitrary Address where
   arbitrary = Address . fromString <$> elements ["DEADBeef", "123456"]
 
+-- | Whether an address is valid or not.
+newtype AddressValidity = AddressValidity { getValid :: Bool }
```

Comment:
What about renaming `getValid` to either `isValid` or `valid`?

Reply:
This was one of the conventions in Serokell where they say you should use getX in newtype constructors. I agree with you though that isValid makes more sense.